Makefile

```diff
@@ -450,11 +450,33 @@ build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION):
 	$(GO) install github.com/golang/protobuf/protoc-gen-go
 	mv $(GOPATH)/bin/protoc-gen-go$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION)
 
-all-protos: internal/pb/backend.pb.go internal/pb/frontend.pb.go internal/pb/function.pb.go internal/pb/messages.pb.go internal/pb/mmlogic.pb.go mmlogic-simple-protos
+all-protos: golang-protos mmlogic-simple-protos php-protos
+golang-protos: internal/pb/backend.pb.go internal/pb/frontend.pb.go internal/pb/function.pb.go internal/pb/messages.pb.go internal/pb/mmlogic.pb.go
 internal/pb/%.pb.go: api/protobuf-spec/%.proto build/toolchain/bin/protoc$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION)
 	$(PROTOC) $< \
-	-I $(CURDIR) -I $(PROTOC_INCLUDES) \
-	--go_out=plugins=grpc:$(GO_SRC)
+		-I $(CURDIR) -I $(PROTOC_INCLUDES) \
+		--go_out=plugins=grpc:$(GO_SRC)
+
+php-protos: examples/functions/php/mmlogic-simple/proto/
+examples/functions/php/mmlogic-simple/proto/:
+	mkdir -p examples/functions/php/mmlogic-simple/proto/
+	$(PROTOC) api/protobuf-spec/messages.proto \
+		-I $(CURDIR) -I $(PROTOC_INCLUDES) \
+		--php_out=examples/functions/php/mmlogic-simple/proto/ \
+		--grpc_out=examples/functions/php/mmlogic-simple/proto/ \
+		--plugin=protoc-gen-grpc=build/toolchain/bin/grpc_php_plugin
```

Comment:
It seems like this line introduced some changes that breaks the make all-protos command. When I tried to build OM on my workstation, the build failed since it Makefile is unable to find the grpc_php_plugin